.env.example

```diff
@@ -0,0 +1,8 @@
+NEXT_PUBLIC_SUPABASE_URL='https://qyrbkygyutnailmwxdum.supabase.co'
+NEXT_PUBLIC_SUPABASE_ANON_KEY='KEY'
+
+# 1. rename to .env.local
+# 2. use this key: eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJzdXBhYmFzZSIsInJlZiI6InF5cmJreWd5dXRuYWlsbXd4ZHVtIiwicm9sZSI6ImFub24iLCJpYXQiOjE3MTg3MjE0ODksImV4cCI6MjAzNDI5NzQ4OX0.wh64VkDdoig1gMMcq3VLKq4hd6-YuYTE-0b7cirN3jY
```

Comment:
3 lines later:
> it's good practice to hide the secrets
and here it is not hidden anyway :/

Reply:
I thought that commenting out may be enough to hide it from automatic detection (and GitGuardian was not triggered). 

What do you suggest, use the power of git to rewrite history, or nuke the secret and use a new one in a secure way?

Reply:
Revoke the key and make a new one that everyone will grab from the Supabase account